recipes/onnxconverter-common/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "onnxconverter-common" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d3b468b19e4318983635e4cda95985c11a554c2e5704240c513ff0c7d9e361bf
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - protobuf
+    - onnx
+
+test:
+  imports:
+    - unittest
```

Comment:
This isn't testing the package; it's just importing `unittest`. You should replace `unittest` with `onnxconverter-common`.